Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [workspace]
 
-members = [
+members = [ "cli",
```

Comment:
I note you added `cli` to `members` but not to `default-members`, which means tests won't be exercised on a simple `cargo test` from the root of the repository. Is this Probably something we should change?

Reply:
There are not tests at the moment in the CLI